src/Symfony/Component/Mime/Security/SMimeSigner.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
We're not doing this in Symfony :)

Reply:
still need to clean-up all the CS and structure 😄